### PR TITLE
Fix issue 1183: add indication of active workspace to project outline

### PR DESCRIPTION
**Pull request by SchweizS**
Changes:
- Added indication of active workspace
- added icon to select active workspace in project outline

Todo:
- localization

#1183

### CODE REVIEW

**Comment by andreeis** [comment]
The icon might not be the most relevant for the "change active workspace" operation. We would prefer 'gear' first, then if not possible the 'utility' icon. If you already tried other icons but got blocked in any way so you had to pick as a last resort the current (build like) icon then let us know and we will try to figure out. If you didn't try yet, please push another commit using one of the two recommended icons.

**Comment by SchweizS** [comment]
i switched it to gear.
i just copy&pasted that part and forgot to change the icon to a better one.

File: src/extension.ts [hunk 1]
```diff
@@ -812,7 +813,10 @@ class ExtensionManager implements vscode.Disposable {
     }
     vscode.window.showErrorMessage(localize('compilation information.not.found', 'Unable to find compilation information for this file'));
   }
-
+  async selectWorkspace(file?: vscode.WorkspaceFolder) {
```
Comment (bobbrow) [inline src/extension.ts:816]: Sorry, minor nitpick.  Can you call this `folder` instead of `file`?

**Comment by SchweizS** [comment]
localize('workspace.active', 'Active Workspace');
localize('workspace', 'Workspace');

those to keys should probably be added to the localization  (tree.ts lines 461/463)

**Pull request merged**